pom.xml

```diff
@@ -204,6 +204,7 @@
 		<dependency>
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2-algorithm</artifactId>
+			<version>0.5.1-SNAPSHOT</version>
```

Comment:
We have to wait until imglib2-algorithm was released or change this back if it has already been released.